docs/readmes/orc8r/deploy_build.md

```diff
@@ -79,6 +79,26 @@ COMPOSE_PROJECT_NAME=magmalte ${PUBLISH} -r ${REGISTRY} -i magmalte -v ${MAGMA_T
 ```
 
 ## Build and publish Helm charts
+*Only Choose one option below*
+### First option: Publish to artifactory
```

Comment:
```suggestion
## Build and publish Helm charts

NOTE: only choose one of the below options

### Option 1: Publish to artifactory
```

Reply:
Actually, these two sections are confusing, because actually step 1 is optional and step 2 is required. So reword this or something to make things clearer